torch/serialization.py

```diff
@@ -1187,8 +1210,11 @@ def _load(zip_file, map_location, pickle_module, pickle_file='data.pkl', **pickl
 
     def load_tensor(dtype, numel, key, location):
         name = f'data/{key}'
-
-        storage = zip_file.get_storage_from_record(name, numel, torch.UntypedStorage)._typed_storage()._untyped_storage
+        if mmap:
```

Comment:
nit: I'm not sure about this here. This file doesn't really care if the overall_storage was mmaped or not. It could come from somewhere else.
I would just `if overall_storage is not None` here.

Reply:
hm mmap here is `overall_storage is not None` on 1199  :b is it bad to save this as an extra variable